Topics/Tech_Stacks/Nodemailer.md

```diff
@@ -0,0 +1,93 @@
+# Nodemailer
+
+## Table of contents
+### [Introduction](#introduction-1)
+### [Create a Transporter object](#create-a-transporter-object-1)
+### [Create a MailOptions Object](#create-a-mailoptions-object-1)
+### [Use the Transporter.sendMail method](#use-the-transportersendmail-method-1)
+
+## Introduction
+
+Nodemailer is a Node.js module that allows you to send emails from your server easily. You can use this Nodemailer however you like. For example communicating with others and notifying yourself if there is any problem. As the name suggests, Nodemailer is use in Node.js, so when you want to use Nodemailer to send mail, mnake sure to use Node.js and have Node.js ready with you.
+
+Before I start explaning how Nodemailer can be used, we have to install and import so we can use it. You can install nodemailer by the below command:
+
+```
+npm install nodemailer
+```
+After you installed Nodemailer, make sure you import nodemailer so you can use it
+
+``` JavaScript
+const nodemailer = require('nodemailer');
+```
+
+[This link](https://www.freecodecamp.org/news/use-nodemailer-to-send-emails-from-your-node-js-server/) provides more practical and detailed mathod of installing necessary package and examples.
```

Comment:
"provides more practical and detailed mathod of installing necessary package and examples." -> "provides a more practical and detailed method of installing the necessary packages, as well as nodemailer examples."